### ISSUE TITLE
asic miner does not connect 

### ISSUE THREAD

**Issue by ctayiz** [alephium/alephium#1197]
hello,

I would like to run an alephium solo mining pool. I have rented an Ubuntu root server. I downloaded the current .jar file and followed the Alephium documentation.
Link: https://docs.alephium.org/full-node/getting-started

My user.conf file looks like this:

alephium.network.external-address = “152.53.111.79:20973”
alephium.mining.miner-addresses = [
  “XXX”,
  “XXX”,
  “XXX”,
  “XXX”
]

alephium.network.rest-port = 22973
alephium.network.ws-port = 21973
alephium.network.miner-api-port = 20973
alephium.api.network-interface = “0.0.0.0”
alephium.api.api-key = “XXXXXXXX”
#alephium.api.api-key-enabled = false
alephium.mining.api-interface = “0.0.0.0”
alephium.network.bind-address = “0.0.0.0:19973”
alephium.network.internal-address = “alephium:19973”
alephium.network.coordinator-address = “alephium:19973”

The problem now is that I cannot connect my ASIC miner (Iceriver AL0) to the pool. 
The node is synced and I see in the logs that the miner connects to the pool.

I hope you can help me. 

If more information is needed I will provide it.

Thanks.

**Comment by agennu** [comment]
You have set up a node that works on the alphstratum protocol. ASICs do not know this protocol, they work on the stratum protocol. I am struggling with this problem myself, I need to run a pool together with the node. I am not a developer, maybe I am wrong about something.

**Comment by goodthebest** [comment]
I've a stratum mining pool solution for ALPH to mine with ASICs such as Iceriver ALPH AL0, you may get in touch with me.

**Comment by agennu** [comment]
> @ctayiz @agennu I've a stratum mining pool solution for ALPH to mine with ASICs such as Iceriver ALPH AL0, you may get in touch with me, if you have not been able to get it done yet.

Hi, I have already solved this problem using miningcore+WebUI.

**Comment by polarker** [comment]
Right, stratum protocol is needed for the miner to communicate with the full node.

**Comment by ctayiz** [comment]
> > @ctayiz @agennu I've a stratum mining pool solution for ALPH to mine with ASICs such as Iceriver ALPH AL0, you may get in touch with me, if you have not been able to get it done yet.
> 
> 
> 
> Hi, I have already solved this problem using miningcore+WebUI.

Hi, 
Nice work! 
Can you show me your config.json for Alephium, please? 

I failed because of that.

**Comment by agennu** [comment]
I'm currently cloning the disk to an SSD, I'll show you later.

**Comment by adrianxp** [comment]
Hi, @agennu, could you help me? How can I contact you?
